pkg/ffuf/config.go

```diff
@@ -54,6 +55,7 @@ func NewConfig(ctx context.Context) Config {
 	conf.StopOnErrors = false
 	conf.StopOnAll = false
 	conf.FollowRedirects = false
+	conf.AltOutput = false
```

Comment:
`AltOutput` doesn't exist in the config struct anymore, so this fails to compile.
```suggestion
```